### PR TITLE
#396 added max width on logos

### PR DESCRIPTION
**Pull request by gustavorf**
## What issue(s) does this solve?

- [x] Issue Number: #WRIN/396

## What is the new behavior?

- Added max width on logos in the sidebar.
-
-

## Profile requirements:

- Does deploying this change require a change to config at the site level (choose one)?
  - [x] No config change is required.
  - [ ] Yes, and I have written an update hook to apply these config changes.
  - [ ] Yes, and I've included updating instructions to be added to the release notes. The next release will need to be a major version increase. (Only do this in special cases.)

## Site-level pull requests for testing. Only merge when these PRs are approved:

Create or update any site-level pull requests following [the documentation](https://github.com/wri/WRIN/wiki/WRI-Dev-Workflow-(Thinkshout)#generating-a-multidev-for-wri_sites-work-review)

- [ ] Flagship PR:

## Checked on develop (TA to do)

- [ ] Brasil
- [ ] China
- [ ] Indonesia

### CODE REVIEW

**Review by mariacha — CHANGES_REQUESTED** [review]
This looks good, but there's also a second request in the ticket:

> Update the code for the logo to not show on subpages, like https://www.wri.org/cities/about. These will have a class of `subpage` that you can target.

Make sure the image does not show on pages like /initiatives/allied-climate-transformation-act2025/resources. You can just hide it with css.

File: themes/custom/ts_wrin/sass/components/_project-detail.scss [hunk 1]
```diff
@@ -47,6 +47,8 @@
       img {
         width: 100%;
         height: auto;
+        max-width: 250px;
```
Comment (mariacha) [inline themes/custom/ts_wrin/sass/components/_project-detail.scss:50]: Now that I see this, can you move this max-width down to the new rule you've made below? I worry that some of the other rules in this list (`landing-page__content` and `.publication__image`) may not want this max-width rule

**Review by mariacha — CHANGES_REQUESTED** [review]
Nice job! One small fix from me

File: themes/custom/ts_wrin/sass/components/_project-detail.scss (hunk 1 above)
Comment (gustavorf) [reply themes/custom/ts_wrin/sass/components/_project-detail.scss]: fixed

**Comment by mariacha** [comment]
Great!

**Pull request merged**